cli/cmd/compliance_gcp.go

```diff
@@ -242,3 +249,57 @@ func complianceGcpReportDetailsTable(report *api.ComplianceGcpReport) [][]string
 		[]string{"Report Time", report.ReportTime.UTC().Format(time.RFC3339)},
 	}
 }
+
+// ALLY-431 Workaround to split the Project ID and Project Alias
+// ultimately, we need to fix this in the API response
+func fixGcpProjectsApiResponse(gcpInfo api.CompGcpProjects) cliComplianceGcpInfo {
```

Comment:
Nitpick: Can we change the function name to be more descriptive. Not a fan of using the word "fix" here.

Reply:
Any suggestion?

Reply:
Nothing great springs to mind. Think I'm ok with fix, I can't think of a better alternative. The argument I was considering was whether we're "fixing" something or not. The api response is what is intended for it's original purpose, even if it's a strange behaviour.

Reply:
I would call it what it does: "**split**GcpProjectsApiResponse"

Reply:
Done! Thank you for your review and suggestion.